packages/elements/src/components/ino-menu/MDCCustomMenu.js

```diff
@@ -3,12 +3,19 @@ import { MDCMenu } from '@material/menu';
 export class MDCCustomMenu extends MDCMenu {
 
   // Really hacky
-  // TODO: Test if basic onclick and keydown events still come trough
```

Comment:
I move this issue to a separate ticket because I think it's not that trivial and will require some work:
https://github.com/inovex/elements/issues/270